releases/EXCEPTIONS.md

```diff
@@ -38,11 +38,17 @@ Email them to:
 
 [You should have *very high confidence* on the “additional time needed” number - we will not grant multiple exceptions for a enhancement.]
 
-Requests for exceptions must be submitted before the first milestone burn down meeting. All requests for exception will be reviewed and either approved or rejected during the first meeting.
+
+Exceptions after enhancements freeze:
```

Comment:
I would suggest delineating this before/after stuff as headings, move it up top since people looking at this doc _now_ are going to be looking at it with enhancements in mind